src/wallet/wallet2.cpp

```diff
@@ -1764,7 +1773,7 @@ void wallet2::refresh(uint64_t start_height, uint64_t & blocks_fetched, bool& re
   {
     // If stop() is called we don't need to check pending transactions
     if(m_run.load(std::memory_order_relaxed))
-      update_pool_state();
+      update_pool_state(refreshed);
```

Comment:
The `refreshed` variable seems necessary because the wallet CLI and RPC call `update_pool_state()` where `refreshed` defaults to `false`.